nuget/Plugin.nuspec

```diff
@@ -62,6 +62,10 @@
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.dll" target="lib\wpa81\Plugin.Media.Abstractions.dll" />
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.xml" target="lib\wpa81\Plugin.Media.Abstractions.xml" />
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.pdb" target="lib\wpa81\Plugin.Media.Abstractions.pdb" />
+     <file src="src\Media.Plugin.WindowsPhone81\bin\Release\CameraCaptureUI.xaml.cs" target="lib\wpa81\CameraCaptureUI.xaml.cs" />
```

Comment:
It seems as these would be adding it as a Library... is this the correct functionality? I assume you actually want it as a file such as: https://github.com/jamesmontemagno/SettingsPlugin/blob/master/nuget/Plugin.nuspec#L43 ???